Ports/rubberband/package.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=rubberband
+version=1.8.1
```

Comment:
Are we sure that we want a 10 year old version instead of the newest?

Reply:
There is no newer version I know of.

Reply:
There is a v3.3.0 on the [official page](https://breakfastquay.com/rubberband/). The official GitHub mirror is [here](https://github.com/breakfastquay/rubberband) (although I'd recommend the official published tarball).

What you are using here seems to be a random third-party mirror/fork.

Reply:
Definitely was an official repo at some point iirc, I'm not sure how i found it though.